python_modules/dagster/dagster_tests/core_tests/test_object_store.py

```diff
@@ -80,6 +90,33 @@ def test_file_system_object_store():
                 pass
 
 
+def test_file_system_object_store_composite_types():
+    run_id = str(uuid.uuid4())
+
+    object_store = FileSystemObjectStore(run_id=run_id)
+    assert object_store.root == os.path.join(
+        seven.get_system_temp_directory(), 'dagster', 'runs', run_id, 'files'
+    )
+
+    with yield_pipeline_execution_context(
+        PipelineDefinition([]), {}, RunConfig(run_id=run_id)
```

Comment:
we should definitely has a common library to capture this sort of pattern (don't need to do it here)

Reply:
or maybe we should there is a lot of this

Reply:
this would be a great first element of a dagster.test_fixtures module